azurefunctions-extension-balo/Ballerina.toml

```diff
@@ -0,0 +1,5 @@
+[project]
+org-name = "ballerinax"
+version = "0.0.0"
```

Comment:
Are we packing this with the distribution? Else this should have a version.

Reply:
Yes, this will go with the distribution.